lstchain/version.py

```diff
@@ -29,6 +29,7 @@
 """
 from subprocess import check_output, CalledProcessError
 from os import path, name, devnull, environ, listdir
+from ast import literal_eval
 
 __all__ = ("get_version",)
```

Comment:
shouldn't this `__all__` contain all the functions defined in the file?

Reply:
I think `get_version` was the only function meant to be part of the API, to be used in `__init__` for example. Actually this file is probably completely private, right? 

So `__all__` does not really have a relevance here.